cross/ti-c6000.txt

```diff
@@ -0,0 +1,22 @@
+# Cross file tested on Texas Instruments C6000 compiler (bare metal DSP devices)
+# This file assumes that path to the Texas Instruments C60000 toolchain is added
+# to the environment(PATH) variable.
```

Comment:
s/environment(PATH)/PATH environment

unless of course other files are following this convention

Reply:
This convention is not used in other files.

Reply:
Please make the edit I suggested then